sample-apps/README.md

```diff
@@ -10,8 +10,6 @@ If you are just wanting to get a demo going, see the [DemoSetup.md](DemoSetup.md
 
 This is the parent directory of several sample applications
 
-**Note: the applications in the sample-apps/ directory generally require java 1.8 or greater.**
```

Comment:
@shreyasdn have a look at this - I added this after your fork. Can you pull in this change?